Corona-Warn-App/src/main/java/de/rki/coronawarnapp/qrcode/ui/QrCodeScannerFragment.kt

```diff
@@ -78,7 +79,7 @@ class QrCodeScannerFragment : Fragment(R.layout.fragment_qrcode_scanner), AutoIn
                     .setOnDismissListener { popBackStack() }
                     .show()
 
-                InProgress -> binding.qrCodeScanSpinner.show()
+                InProgress -> binding.qrCodeProcessingView.isVisible = true
```

Comment:
this line looks redundant to me as you are using
```kotlin
binding.qrCodeProcessingView.isVisible = scannerResult == InProgress
```

Reply:
I know, but `when` is exhaustive if the state is not added, complier complaints